exasol/slc_ci/model/flavor_ci_model.py

```diff
@@ -8,6 +8,7 @@ class TestSet(BaseModel):
     folders: List[str]
     goal: str
     test_languages: List[str]
```

Comment:
Not needed, because generic_language_tests is actually used by exaslct and test_languages is usually.

Reply:
Removed